data-model-layer/data-model.md

```diff
@@ -85,9 +85,42 @@ Boolean is a scalar kind.  Its cardinality is two -- either the value 'true' or
 
 #### Integer kind
 
+Integer is a scalar kind and refers to whole numbers without a fractional
+component.
+
+It is important to consider codec and language limitations that may be imposed
+on the serialization of integers from the IPLD Data Model. For example:
+
+* Some codecs, such as DAG-CBOR, will assume that integers must be within the
+  64-bit signed range and reject anything larger.
+* IPLD libraries, such as go-ipld-prime, limit their in-memory representation
+  of the integer kind to the signed 64-bit range.
+* JavaScript has difficulties safely handling and representing integers outside
+  of the 53-bit unsigned range and differentiating between integers and floats
+	where there is no fractional component.
```

Comment:
mixing tabs and spaces? :)

Reply:
sigh .. the perils of having JS & Go people share docs, and me being a Go+JS person using the same editor for both, will fix